CHANGELOG.md

```diff
@@ -0,0 +1,15 @@
+# Changelog - chai-webdriverio
+
+## Unreleased Changes
+#### Breaking Changes:
+- Changed `there` and `visible` from methods to properties:
```

Comment:
@mltsy TL;DR; because JavaScript...

```js
expect('test').to.be.foobar
```

Is totally fine as long as ```to``` and ```be``` are objects.
The way chai works is that it overrides the property getter when you create a property assertion, but it only does that for property's you specify. 

simplified example of what happens if I created the foobar property assertion:
```js
chai.to.be = Object.create(Object.prototype, {
  foobar: {
    configurable: false,
    get: function() { throw 'I always give errors, because I'm special'; },
});
```
In that example ```to.be.foobar``` will always throw an error, but ```to.be.somethingelse``` will do nothing because that just evaluates as ```undefined```. Function calls will fail because you tried to execute ```undefined``` as an ```function```